src/Services/GitPrMatcherService.php

```
@@ -99,13 +100,13 @@ public function matchAndLink(string $prTitle, string $prBody, string $branch, st
                 )
             );
         } catch (\Throwable $e) {
-            AiWorkflowClaim::release($this->db, $orgId, 'git_pr_match', $claimFingerprint);
+            AiWorkflowClaim::release($this->db, $orgId, 'git_pr_match', $claimFingerprint, $claimLeaseToken);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\AiWorkflowClaim' in method 'matchAndLink'. (undefined)

(StaticAccess)